cdap-api/src/main/java/co/cask/cdap/api/data/stream/StreamWriter.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright © 2015 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.api.data.stream;
+
+import co.cask.cdap.api.stream.StreamEventData;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.util.Map;
+
+/**
+ * Enabling Programs to write to Streams through their Context.
+ */
+public interface StreamWriter {
+
+  /**
+   * Write a string to a stream
```

Comment:
I think for javadoc, all verbs are of third-person singular form (Writes, Sets, Reads).